wikihow2zim/entrypoint.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import argparse
+
+from .constants import NAME, setDebug, getLogger
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog=NAME,
+        description="Scraper to create ZIM files from Kolibri channels",
+    )
+
+    parser.add_argument(
+        "--language",
+        choices=['en', 'de', 'fr'],
+        default='en',
+        help="Choose the language (en, de, fr ...)"
+    )
+
+    parser.add_argument(
+        "--optimization-cache",
+        help="URL with credentials to S3 for using as optimization cache",
+        dest="s3_url_with_credentials",
+    )
+
+    parser.add_argument(
+        "--low-quality",
+        help="Uses only the `low_res` version of videos if available. "
+        "If not, recompresses using agressive compression.",
+        action="store_true",
+        default=False,
+    )
+
+    parser.add_argument(
+        "--no-external-links",
+        help="If specified, don't include external links",
```

Comment:
Don't write `If specified` as it is implied by the use of the param. When writting this on a single param, it may confuse user into thinking there's a special behavior.